scripts/util.py

```diff
@@ -21,6 +21,7 @@
     "Makefile-project-targets",
     "Makefile-project-testing",
     ".gitignore",
+    "run-cbmc-proofs.py",
```

Comment:
Are users expected to modify the Python script?
If so, I think we should add a brief comment regarding what's beyond the scope of the command-line flags and might require modifications within the script.